client_interface.py

```diff
@@ -0,0 +1,74 @@
+"""
+Client which send messages to server
+"""
+import asyncio
+import sys
+import aiohttp
+import json
+import pyglet
+
+from interface_frontend import draw_interface, create_window, handle_text
+from interface import get_interface_state
+
+
+state = get_interface_state()
+window = create_window()
+
+ws = None
+
+@window.event
+def on_draw():
+    """
+    Draw the interface state.
+    """
+    window.clear()
+    draw_interface(state, window)
+
+
+@window.event
+def on_text(text):
+    """
+    Key listener.
+    Wait for user input on keyboard and react for it.
+    """
+    handle_text(state, text)
+    send_to_server(state)
+
+
+def send_to_server(state):
+    """
+    Client send selected cards to server.
+    """
+    msg = json.dumps(state.my_program)
+    print(msg)
```

Comment:
tento `print` je jen pro ověření dat? Nebo má nějaký důležitější význam?

Reply:
jojo jen jsem chtěla kontrolovat ve dvou okýnkách jestli se posílá to stejné. důležitý význam není :-D